### PR TITLE
Edit review of Hosting a Prefect server instance

### PR DESCRIPTION
**Pull request by stellardave**
- Made content clearer
- Used active language when possible
- Made some structural changes
- Fixed typo

### CODE REVIEW

File: docs/guides/host.md [hunk 1]
```diff
@@ -59,11 +65,11 @@ prefect config set PREFECT_API_URL="http://127.0.0.1:4200/api"
 !!! tip "`PREFECT_API_URL` required when running Prefect inside a container"
     You must set the API server address to use Prefect within a container, such as a Docker container.
 
-    You can save the API server address in a [Prefect profile](/concepts/settings/). Whenever that profile is active, the API endpoint will be be at that address.
+    You can save the API server address in a [Prefect profile](/concepts/settings/). Whenever that profile is active, the API endpoint is at that address.
 
     See [Profiles & Configuration](/concepts/settings/) for more information on profiles and configurable Prefect settings.
 
-## Prefect database
+## About Prefect database
```
Comment (discdiver) [inline docs/guides/host.md:72]:
```suggestion
## The Prefect database
```

Comment (discdiver) [reply docs/guides/host.md:72]: "About Prefect database" sounds strange, but I'm not sure I love "The Prefect database" either.

File: docs/guides/host.md [hunk 2]
```diff
@@ -266,11 +270,11 @@ To configure a notification in a Prefect server, go to the **Notifications** pag
...
 
-Notifications are structured just as you would describe them to someone. You can choose:
+You can choose:
 
-- Which run states should trigger a notification.
-- Tags to filter which flow runs are covered by the notification.
-- Whether to send an email, a Slack message, Microsoft Teams message, or other services.
+- Which run states should trigger a notification
+- Tags to filter which flow runs are covered by the notification
+- Whether to send an email, a Slack message, Microsoft Teams message, or other services
```
Comment (discdiver) [inline docs/guides/host.md:280]:
```suggestion
- Whether to send an email, a Slack message, Microsoft Teams message, or use another services
```

**Review by discdiver — COMMENTED** [review]
Thank you. Minor suggestions only on ths one..
Pushed a few notification block additions.

File: docs/guides/host.md (hunk 2 above)
Comment (stellardave) [reply docs/guides/host.md:280]: should be "use another service"

**Comment by stellardave** [comment]
Thanks @discdiver! Just one more tweak on your tweak.

**Review by discdiver — APPROVED** [review]
Thank you!

**Pull request merged**